File: python/7 kyu/38.py
# https://www.codewars.com/kata/5834315e06f227a6ac000099/train/python

# Passed

from collections import Counter

def elimination(arr):
    if not arr:
        return None
    
    counter = Counter(arr)
    twins = counter.most_common(1)[0]
    return twins[0] if twins[-1] > 1 else None

output = elimination([2, 3, 4, 11, 32, 43, 54, 53, 64, 12, 43, 52])
print(output)

File: nodejs/7 kyu/38.js
// https://www.codewars.com/kata/585af8f645376cda59000200/train/javascript

// Passed

function formatPoem(poem) {
  return poem
    .split(".")
    .map((sentence) => sentence.trim())
    .join(".\n")
    .trim();
}

const output = formatPoem(
  "Beautiful is better than ugly. Explicit is better than implicit. Simple is better than complex."
);
console.log(output);